src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -199,7 +199,7 @@ public void MaxRequestContentBufferSize_Set_ThrowsPlatformNotSupportedException(
             }
         }
 
-        [ActiveIssue(20010, TargetFrameworkMonikers.Uap)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "UAP will send default credentials based on other criteria.")]
```

Comment:
No.  It is difficult to write a test for this since it involves using/not-using Enterprise Authentication and other criteria. I have already documented this platform difference in https://github.com/dotnet/corefx/wiki/ApiCompat.